en/drive_client/drive_client_2.0_for_windows_10.md

```diff
@@ -28,14 +30,34 @@ You can control which files or folder are cached locally. This can be changed fr
 
 ![seadrive-cache-control](imgs/seadrive-cache-control.png)
 
-In SeaDrive 1.0, cached files are not automatically updated when they're updated on the server. In SeaDrive 2.0, full and pinned files are automatically kept in sync with the server.
+In SeaDrive 1.0, cached files are not automatically updated when they're updated on the server. In SeaDrive 3.0, full and pinned files are automatically kept in sync with the server.
 
 ## File Download and Control
 
 Whenever you open a placeholder file, the operating system will automatically start to download it. If the file may take some time to download, there will be a progress bar shown up in file explorer and you may cancel the download.
 
 Sometimes a background application may try to download a file in the virtual drive (such as an Anti-Virus software). You will be notified by the operating system about this and you may choose to cancel the download or disallow the application from automatically downloading files in the future.
 
+## Cache Management
+
+You can customize the cache directory and set the cache size limit and clean interval in the settings.
+
+When the cache size surpasses the limit, SeaDrive will automatically free up space by prioritizing files based on their last modification time. This process continues until the cache size is reduced to below 70% of the specified limit.
```

Comment:
When the cache size exceeds the limit, SeaDrive will automatically free up space based on files' last modification time. Older files will be cleaned up first. This process continues until the cache size is reduced to below 70% of the specified limit.